.github/workflows/sign.yml

```diff
@@ -34,162 +26,38 @@ jobs:
-                ]
-              }
-            }
-          ]
-        }
-        '@
-        $workspace = [regex]::escape("${{github.workspace}}")
-        $betaInputJson = $betaInputJsonTemplate -replace '@@workspace@@', $workspace
-        Out-File -FilePath .\betaInput.json -InputObject $betaInputJson
-
-    - name: Create Stable Input File
-      run: |
-        $stableInputJsonTemplate = @'
-        {
-          "Version": "1.0.0",
-          "SignBatches": [
-            {
-              "SourceLocationType": "UNC",
-              "SourceRootDirectory": "@@workspace@@\\unsigned\\stable",
-              "DestinationLocationType": "UNC",
-              "DestinationRootDirectory": "@@workspace@@\\signed\\stable",
-              "SignRequestFiles": [
-                {
-                  "SourceLocation": "${{inputs.assembly_name}}.${{inputs.stable_version}}.nupkg",
-                  "DestinationLocation": "${{inputs.assembly_name}}.${{inputs.stable_version}}.nupkg"
-                }
-              ],
-              "SigningInfo": {
-                "Operations": [
-                  {
-                    "KeyCode": "CP-401405",
-                    "OperationCode": "NuGetSign",
-                    "ToolName": "sign",
-                    "ToolVersion": "1.0"
-                  },
-                  {
-                  "KeyCode": "CP-401405",
-                    "OperationCode": "NuGetVerify",
-                    "ToolName": "sign",
-                    "ToolVersion": "1.0"
-                  }
-                ]
-              }
-            }
-          ]
-        }
-        '@
-        $workspace = [regex]::escape("${{github.workspace}}")
-        $stableInputJson = $stableInputJsonTemplate -replace '@@workspace@@', $workspace
-        Out-File -FilePath .\stableInput.json -InputObject $stableInputJson
+    - name: Setup NuGet
+      uses: NuGet/setup-nuget@v1.0.5
 
     - name: Azure Login
       uses: azure/login@v1
       with:
           creds: ${{ secrets.az_creds }}
 
-    - name: Install ESRP Client
+    - name: Run Signing Script
       run: |
-        az storage blob download --auth-mode login --subscription  ${{ secrets.az_sub }} --account-name ${{ secrets.storage_name }} -c ${{ secrets.container_name }} -n microsoft.esrpclient.1.2.76.zip -f esrp.zip
-        unzip -d ./esrp esrp.zip
-
-    - name: Install Certificates
-      run: |
-        az keyvault secret download --subscription "${{ secrets.az_sub }}" --vault-name "${{ secrets.vault_name }}" --name "${{ secrets.aad_cert }}" -f cert.pfx
-        certutil -silent -f -importpfx cert.pfx
-        rm cert.pfx
-        az keyvault secret download --subscription "${{ secrets.az_sub }}" --vault-name "${{ secrets.vault_name }}" --name "${{ secrets.sign_cert }}" -f cert.pfx
-        certutil -silent -f -importpfx cert.pfx
-        rm cert.pfx
+        $url = "https://raw.githubusercontent.com/microsoft/digitalworkplace-workflows/users/mibir/new-workflows/scripts/sign.ps1"
```

Comment:
I suppose you'll update this url to use the main branch's file before merging the PR?

Reply:
Correct